ADR/user-specified-tokens-build.md

```diff
@@ -0,0 +1,75 @@
+# Enable users to configure tokens/credentials for a Build/Test Pipeline
+
+Date: 2023-03-03
+
+## Status
+
+Provisional
+
+
+## Context
+
+As a user, I would like to configure sensitive information/tokens/credentials for the build/test pipeline to consume as part of a partner Task.
+
+## What is a Partner Task
+
+A Partner Task is a Tekton Task that is provided and maintained by a Red Hat Partner. However, this initiative also aims to enable Tekton Tasks 
+contributed to the StoneSoup Build/Test Pipeline by different organizations inside Red Hat.
+
+
+## Decision
+
+### How would the sensitive information be persisted and made available to the PipelineRun ?
+
+1. The UI would take in sensitive information as form input and upload it to SPI using the user's authentication token. See https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/docs/USER.md#uploading-access-token-to-spi-using-kubernetes-secret .
+
+2. Subsequently, the UI would create an `SPIAccessTokenBinding` CR with the appropriate label in the `.spec.secret.labels` to make the `Secrets` available to the build service or the integration service.
```

Comment:
Can you revise the text to give an example of the appropriate label?

Here, do you mean the `tekton.dev/...` annotations (not labels) that when they appear on a `Secret`, cause tekton to perform credential initialization? https://github.com/tektoncd/pipeline/blob/main/docs/auth.md#understanding-credential-selection

Reply:
Oh, no, I see now. I think you intend for this to be a custom labeling (like `appstudio.redhat.com/<parameter-name>` +/- the component's name), to be interpreted by our controllers (build-service and integration-service).